LA4Ice.c

```diff
@@ -283,7 +283,7 @@ int main(int argc, char *argv[])
   }
 
   //  Read the file and display selected records
-  
+  if (tspace > 0)
```

Comment:
if tspace is 0, then line 328 would raise a seg fault, that's what I am trying to avoid. 

328        tp_wide = Number_Digits((int64) db1->maxlen/tspace+2);